README.md

```diff
@@ -108,8 +108,37 @@ dev.ror.192.168.1.5.xip.io or ror.192.168.1.5.xip.io
 
 ## Remote Debugging
```

Comment:
should follow this best practices http://dev.teracy.org/docs/nodejs_dev_guide.html#remote-debugging